service/src/main/resources/application.yaml

```diff
@@ -57,6 +60,7 @@ ccd:
       url: http://fake-url
 
 fpl:
+  local_authority_codes_forbidden_case_submission: FPLA
```

Comment:
Probably it works. Suprised you don't need - to make it a list.

Reply:
Since it is declared as `List<String>` it does split on `,` with some whitespace trimming.

Reply:
Oh, that's clever. Thanks I should check the docs!